docs/user-guide/timeouts/timeouts-theory.md

```diff
@@ -0,0 +1,62 @@
+# Timeouts in Certora Prover -- Theory Background
```

Comment:
This section seems to be interesting, but also too advanced and not really talking about how to user our tool. @mdgeorge4153, is this a good place for such information?